src/mysql_vm_helpers.py

```diff
@@ -257,7 +257,7 @@ def setup_logrotate_and_cron(self) -> None:
         with open("/etc/logrotate.d/flush_mysql_logs", "w") as file:
             file.write(rendered)
 
-        cron = "* * * * * root logrotate -f /etc/logrotate.d/flush_mysql_logs\n"
+        cron = "* 1-23 * * * root logrotate -f /etc/logrotate.d/flush_mysql_logs\n1-59 0 * * * root logrotate -f /etc/logrotate.d/flush_mysql_logs\n"
```

Comment:
nit:
```suggestion
        cron = (
            "* 1-23 * * * root logrotate -f /etc/logrotate.d/flush_mysql_logs\n"
            "1-59 0 * * * root logrotate -f /etc/logrotate.d/flush_mysql_logs\n"
        )
```

Reply:
addressed in [eea60e1](https://github.com/canonical/mysql-operator/pull/365/commits/eea60e1a82000e4218b7fd04f1340cd8b46abeca)